modules/rostests/apitests/browseui/IACLHistory.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
+ * PURPOSE:     Test for IACLHistory objects
+ * COPYRIGHT:   Copyright 2021 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#define _UNICODE
+#define UNICODE
+#include <apitest.h>
+#include <shlobj.h>
+#include <atlbase.h>
+#include <atlcom.h>
+#include <stdio.h>
+#include <shellutils.h>
+
+struct CCoInit
```

Comment:
Might be worth moving this to `apitest.h` or a common header like that?
(Does not have to be done in this PR, but something to keep in mind)